### ISSUE TITLE
Delete/Store Successful Form Submissions 

### ISSUE THREAD

**Issue by shaunburdick** [department-of-veterans-affairs/va.gov-team#84348]
## Value Statement

**_As a_** Veteran

**_I want to_** my submission data to be removed when no longer needed

**_So that_** my privacy is protected

---

## Background Context

- Once we implement #84329, we need to make sure we delete any data we no longer need in accordance with our ATO

## Outcome, Success Measure, KPI(S), and Tracking Link

- Any data that is no longer needed is removed from the database
- Longer term, more secure storage may need population

## Acceptance Criteria

- [x] Any submission data is removed once the 60 day window is complete

## Designs and Build Notes

- Add here

### Enablement team (if needed)
- @va-albers 
- @CorySohrakoffUSDS 

## Out of scope

- Creation of the tables, it's handed in #84329
- Polling/Status determination, it's handled in #84332

## Open questions

- Is there a need for longer term, more secure storage of submission data?

## Tasks

- [x] Create a process that will scan for submissions that need to be deleted
- [x] Delete that data
- [ ] Possibly archive in secure long-term storage

## Definition of Ready

- [ ] Clear value description
- [ ] Testable acceptance criteria
- [ ] Accessibility added to acceptance criteria
- [ ] Approved designs attached
- [ ] Sample data provided where appropriate
- [ ] Estimated to fit within the sprint
- [ ] Dependencies and blockers linked

## Definition of Done

- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Reviewed and approved by product and/or design

**Comment by dfong-adh** [comment]
PR ready for review: https://github.com/department-of-veterans-affairs/vets-api/pull/17774

**Comment by anniebtran** [comment]
Will need more code reviews

**Comment by dfong-adh** [comment]
Verfied in staging, enabling in prod today

**Comment by dfong-adh** [comment]
Flag `decision_review_delete_saved_claims_job_enabled` enabled in prod.
